src/codeu/chat/client/commandline/Chat.java

```diff
@@ -46,15 +50,23 @@ public Chat(Context context) {
   // the system wants to exit, the function will return false.
   //
   public boolean handleCommand(String line) {
+    final Tokenizer tokens;
+    final String command;
 
-    final Scanner tokens = new Scanner(line.trim());
-
-    final String command = tokens.hasNext() ? tokens.next() : "";
+    try {
+      tokens = new Tokenizer(line.trim());
+      command = tokens.hasNext() ? tokens.next() : "";
+    }
+    catch (IllegalArgumentException e) {
+      // Catch any misformatting or unclear input here and tell user
```

Comment:
Combine the two comments into one line.  Your content should be describing the intent behind a block of code, rather line by line.